README.md

```diff
@@ -276,15 +276,15 @@ ansible-playbook sap+hana-assert.yml -l host_1,host_2 |
 awk '{sub ("    \"msg\": ", "")}
   /TASK/{task_line=$0}
   /fatal:/{fatal_line=$0; nfatal[host]++}
-  /...ignoring/{nignore[host]++}
+  /...ignoring/{nfatal[host]--; if (nfatal[host]<0) nfatal[host]=0}
```

Comment:
Is it possible for a task to report `...ignoring` when the task status is not `fatal`?

Reply:
It is in task "Assert that all required packages are installed" in tasks/RedHat/assert-installation.yml. This task is reporting either "failed:" in case a package is not installed or "ok:" in case a package is installed.
This task has set ignore_errors to the content of a variable. If this variable is true, and if one package is missing, then the task is reporting "...ignoring" without a "fatal:". If no package is missing, there will be no "...ignoring" message.